src/components/Text/__test__/Text.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Text } from '..';
+
+describe('Text component', () => {
+    it('renders text content', () => {
+        render(<Text>Test Text</Text>);
+        expect(screen.getByText('Test Text')).toBeInTheDocument();
+    });
+
+    it('renders with default props', () => {
+        render(<Text>Default Text</Text>);
+        const defaultText = screen.getByText('Default Text');
+        expect(defaultText).toHaveClass("derivs-text__weight--normal")
+
+    });
```

Comment:
we can merge these two test cases:
```suggestion
    it('renders with default props', () => {
        render(<Text>Test Text</Text>);
        const defaultText = screen.getByText('Test Text');
        expect(defaultText).toBeInTheDocument();
        expect(defaultText).toHaveClass("derivs-text__weight--normal")
    });
```